lib/v2/gov/nrta/dsj.js

```diff
@@ -0,0 +1,59 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const asyncPool = require('tiny-async-pool');
+
+module.exports = async (ctx) => {
+    const { category = 'note' } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 2;
+
+    const rootUrl = 'https://dsj.nrta.gov.cn';
+    const currentUrl = new URL(`tims/site/views/applications.shanty?appName=${category}`, rootUrl).href;
+
+    const { data: response } = await got(currentUrl);
+
+    const $ = cheerio.load(response);
+
+    const items = $('img[src="/site/styles/default/images/icon_arrow_r.gif"]')
+        .slice(0, limit)
+        .toArray()
+        .map((item) => {
+            item = $(item).next();
+
+            const pubDateMatches = item.text().match(/(\d+年\d+月)/);
+
+            return {
+                title: item.text(),
+                link: new URL(item.prop('href'), rootUrl).href,
+                pubDate: pubDateMatches ? parseDate(pubDateMatches[1], ['YYYY年MM月', 'YYYY年M月']) : undefined,
+            };
+        });
+
+    const results = [];
+
+    for await (const item of asyncPool(1, items, (item) =>
```

Comment:
From my testing, I would guess the safe boundary is 5 for the total of 15.

Reply:
The result of my test here is that the safe boundary is barely 2. 😅 Let's give it a shot for 5.